### PR TITLE
fix: Typography.Paragraph\Text\Title ts

### LINKED ISSUE #29772: Typography 的 types 定义有问题，onClick 等方法提示不存在
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/antd-reproduction-template-forked-7wqtx?file=/Text.tsx)

### Steps to reproduce
`Typography.Text` 组件调用 `onClick` 方法 typescript 提示 `onClick` 不存在，但实际调用可以使用

### What is expected?
typescript 提示正常

### What is actually happening?
```
Type '{ children: string; onClick: () => void; }' is not assignable to type 'IntrinsicAttributes & TextProps & { children?: ReactNode; }'.
  Property 'onClick' does not exist on type 'IntrinsicAttributes & TextProps & { children?: ReactNode; }'.ts(2322)
```

| Environment | Info |
|---|---|
| antd | 4.13.1 |
| React | 17.0.1 |
| System | macOS |
| Browser | Chrome |

### PR DESCRIPTION
**Pull request by bingling0084**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [x] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/29772

### 💡 Background and solution
修复 Typography ts 定义：Typography.Paragraph、Typography.Text、Typography.Title 添加 onClick，更改 doc

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Update Typography types and doc.   |
| 🇨🇳 Chinese |  更新 Typography.Paragraph\Text\Title 类型定义以及文档。  |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/Paragraph.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
 import Base, { BlockProps } from './Base';
 
-export interface ParagraphProps extends BlockProps {}
+export interface ParagraphProps extends BlockProps {
```
Comment (xrkffgg) [inline components/typography/Paragraph.tsx:4]: 直接在 extends 里加就行吧。 md 中 API 排下序哈~

Comment (bingling0084) [reply components/typography/Paragraph.tsx:4]: 现在的顺序哪里有问题吗，API 排序是什么顺序排呢？

Comment (xrkffgg) [reply components/typography/Paragraph.tsx]: 字母排序

Comment (bingling0084) [reply components/typography/Paragraph.tsx]: 还是不要加到 extends 吧，BlockProps 被引用的地方比较多，比如 getEllipsis props 没必要加上 onClick。然后API 已经是按字典序排的了呀

Comment (xrkffgg) [reply components/typography/Paragraph.tsx:4]: o 在 s 前

Comment (bingling0084) [reply components/typography/Paragraph.tsx:4]: 重排了序，还发现个问题，Paragraph\Text\Title  实际上并不支持 onChange，先从文档中去掉了～

**Pull request merged**